main_server.py

```diff
@@ -321,7 +343,9 @@ async def update_core_config(request: Request):
 
 @app.on_event("startup")
 async def startup_event():
-    global sync_process
+    global sync_process, registered_voices  # 添加registered_voices
```

Comment:
```suggestion
    global sync_process
```

Again, don't use global variable